app/routes.py

```diff
@@ -0,0 +1,177 @@
+import re
+import logging
+import time
+
+from flask import abort
+from flask import jsonify
+from flask import make_response
+from flask import request
+from flask import redirect
+from werkzeug.exceptions import BadRequest
+
+from app import app
+from app.helpers import add_item
+from app.helpers import fetch_url
+from app.helpers.checks import check_params
+from app.helpers.response_generation import make_error_msg
+from app.models.dynamo_db import get_dynamodb_table
+from service_config import Config
+
+logger = logging.getLogger(__name__)
+
+base_response_headers = {
+    'Content-Type': 'application/json; charset=utf-8',
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'GET, OPTION',
+    'Access-Control-Allow-Headers': 'Content-Type, Authorization, x-requested-with, Origin, Accept'
+}
+
+
+@app.route('/checker', methods=['GET'])
+def checker():
+    """
+    * Quick summary of the function *
+
+    Simple function to check if the service is alive with no call to any other functions.
+
+    * Abortions originating in this function *
+
+    None
+
+    * Abortions originating in functions called from this function *
+
+    None
+
+    * Parameters and return values *
+
+    :return: a simple json saying basically 'OK'
+    """
+    logger.info("Checker route entered at %f", str(time.time()))
```

Comment:
Must be here definitive a debug log ! The checker is called very often !

Reply:
I'm changing all "entry in function" logs to debug.